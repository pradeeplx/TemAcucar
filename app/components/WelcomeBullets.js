import React from 'react'
import { View, StyleSheet } from 'react-native'
import { fontFactor } from "../helpers"
import Colors from "../Colors"

export default WelcomeBullets = ({ active }) => (
  <View style={{
    flexDirection: 'row',
    marginVertical: 20,
  }}>
    { [1, 2, 3, 4, 5].map((index) => (
      <View key={index} style={{
        backgroundColor: (active === index ? Colors.blue : Colors.white),
        borderColor: (active === index ? Colors.blue : Colors.gray),
        borderWidth: StyleSheet.hairlineWidth,
        width: 8 * fontFactor(),
        height: 8 * fontFactor(),
        borderRadius: 4 * fontFactor(),
        marginHorizontal: 4 * fontFactor(),
      }}/>
    )) }
  </View>
)
