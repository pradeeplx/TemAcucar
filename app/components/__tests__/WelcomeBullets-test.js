import 'react-native'
import React from 'react'
import WelcomeBullets from '../WelcomeBullets'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <WelcomeBullets active={3} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
