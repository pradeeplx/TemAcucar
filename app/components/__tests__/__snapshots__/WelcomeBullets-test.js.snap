exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "marginVertical": 20,
    }
  }>
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "borderColor": "#CECED2",
        "borderRadius": 6,
        "borderWidth": 0.5,
        "height": 12,
        "marginHorizontal": 6,
        "width": 12,
      }
    } />
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "borderColor": "#CECED2",
        "borderRadius": 6,
        "borderWidth": 0.5,
        "height": 12,
        "marginHorizontal": 6,
        "width": 12,
      }
    } />
  <View
    style={
      Object {
        "backgroundColor": "#44B5B1",
        "borderColor": "#44B5B1",
        "borderRadius": 6,
        "borderWidth": 0.5,
        "height": 12,
        "marginHorizontal": 6,
        "width": 12,
      }
    } />
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "borderColor": "#CECED2",
        "borderRadius": 6,
        "borderWidth": 0.5,
        "height": 12,
        "marginHorizontal": 6,
        "width": 12,
      }
    } />
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "borderColor": "#CECED2",
        "borderRadius": 6,
        "borderWidth": 0.5,
        "height": 12,
        "marginHorizontal": 6,
        "width": 12,
      }
    } />
</View>
`;
